_posts/national/united-kingdom/united-kingdom.Rmd

```diff
@@ -56,23 +56,42 @@ We calculate Rt as the average of how many new infections arise from one infecte
 To explore in more depth how Rt from different data sources can be used to understand transmission dynamics across the population, and a more detailed methodology, see our pre-print [@rt-comparison].
 
 ```{r rt-comparison-nations, fig.height = 12, fig.width = 14, out.width = "95%", echo=FALSE}
+# Rt
 summary <- suppressMessages(readr::read_csv(here::here("covid-rt-estimates/subnational/united-kingdom/collated/rt/summary_latest.csv"))) %>%
```

Comment:
Can you update this to pull from the local copy rather than the GitHub version? This will allow the code to be robust to results ever moving from GitHub (if that ever happens). Local copy approach looks like it will be something we keep using but pull the results from storage other than GitHub

Reply:
By local copy do you mean the `covid/covid-rt-estimates` folder stored within the website repo? This is where it currently pulls from (ie `here::here("covid-rt-estimates")` with `here::` starting in the `covid/` project folder)

Reply:
ah right yes sorry - for some reason I thought this was a URL. My bad.